apps/paymaster-proxy/src/errors/JsonRpcError.ts

```diff
@@ -0,0 +1,78 @@
+import type { PaymasterRpcError } from '@/errors/PaymasterError'
```

Comment:
This is the error that our proxy service will return. In most cases, it just returns the same error that the underlying service returned.